ae/utils/get_detailed_accessibility_tree.py

```diff
@@ -425,7 +417,7 @@ def __should_prune_node(node: dict[str, Any], only_input_fields: bool):
     if node.get('role') in ['separator']:
         return True
     #check if the node only have name and role, then delete that node
-    if len(node) == 2 and 'name' in node and 'role' in node:
+    if len(node) == 2 and 'name' in node and 'role' in node and node.get('role') != "button":
```

Comment:
does this mean that we will have button nodes with just name and role? How will we properly map them to elements if we do not have mmid?